sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs

```diff
@@ -92,7 +92,7 @@ public static IWebJobsBuilder AddEventHubs(this IWebJobsBuilder builder, Action<
             return builder;
         }
 
-        public static IWebJobsBuilder AddEventHubsScaleForTrigger(this IWebJobsBuilder builder, TriggerMetadata triggerMetadata)
+        internal static IWebJobsBuilder AddEventHubsScaleForTrigger(this IWebJobsBuilder builder, TriggerMetadata triggerMetadata)
```

Comment:
Are you sure this is safe?  I thought the Functions infrastructure was using this.

Reply:
I looked in the scale controller repo and couldn't find any usages. I assume they are not using our dev feed but will add scale controller folks to the thread.

Reply:
Actually it does look like they are consuming an alpha build, but they are pinned to a specific version.